master/buildbot/db/buildslaves.py

```diff
@@ -35,9 +35,30 @@ def findBuildslaveId(self, name):
                 info={},
             ))
 
-    def buildslaveConfigured(self, buildslaveid, buildermasterids):
+    def buildslaveConfigured(self, buildslaveid, masterid, builderids):
```

Comment:
I see that this method isn't documented at all :disappointed:.  Let's fix that here!